src/System.Text.Utf8/src/System/Text/Utf8/Utf8String.CodePointEnumerable.cs

```diff
@@ -46,6 +46,18 @@ public CodePointEnumerator GetEnumerator()
                 }
             }
 
+            public CodePointReverseEnumerator GetReverseEnumerator()
+            {
+                if (_bytes != null)
```

Comment:
I'd need to pin the memory for the byte[]. According to Krzysztof C. this is slow. I already am also making System.Slices which are probably gonna replace this - it is using some IL hacks to solve memory pinning problems. We need to test Slices and perf compare Utf8String with both implementations and check which one is faster. Considering that there is not much to perf test yet I decided that I'm gonna postpone this a little